### ISSUE TITLE
Axios lib gets detected as a trojan

### ISSUE THREAD

**Issue by JaielZeus** [axios/axios#6304]
### Describe the issue

On Windows 11
Axios v1.6.8, but also older like v1.3.x

Windows and Chrome is saying a potential virus is detected in an app that bundles axios.js as a dependency. Chrome after downloading and if you force the download Windows detects it as a threat and quarantines it. I am using Electron + NodeJS and had Axios in the project. I switched to using ``fetch`` now because of my client reporting this issue.

![image](https://github.com/axios/axios/assets/42944799/8ba19ce5-5cd2-4823-b213-b9eed107f900)

**Comment by justindhillon** [comment]
I believe this is a false positive. [This is the website](https://www.microsoft.com/en-us/wdsi/filesubmission) to ask Microsoft to manually check the software. If you have made sure you are using the up to date lib from the official repo, there is nothing else we can do.

**Comment by JaielZeus** [comment]
@justindhillon  Yeah, this is of course a false positive. I jsut wanted to bring that to light. It might affect others as well. My client, it did but not me for example. It's an AI thing apparently because of the ``!ml`` ending for machine-learning.

**Comment by justindhillon** [comment]
I have submitted the file and this is the report from microsoft:

![image](https://github.com/axios/axios/assets/145078271/5256398c-6bfd-4423-9fe5-55a58dc4fd23)

If @JaielZeus could follow the steps and send the logs to https://www.microsoft.com/en-us/wdsi/filesubmission, then we can get to the bottom of this. Otherwise, there is nothing we can do on our part.

**Comment by jasonsaayman** [comment]
seems like a MS thing closing for now

**Comment by JaielZeus** [comment]
Sorry for not following up. Yes this was a windows thing. I remember even by creating a text file with just the content ``<script>`` in it would trigger a virus detection alert. It was fixed ever since and was not related to axios at all.
